generators/bdk/python/templates/readme.md.ejs

```diff
@@ -0,0 +1,17 @@
+# Sample BDK 2.0 for Python project
+This is a sample bot project using the Symphony BDK 2.0 for Python.
+To get started, follow these commands below:
+
+## First run only:
+1. Create virtual environment:
+    - `python3 -m venv env`
```

Comment:
This should be done thanks to the `install()` command, right ?

Reply:
The readme is more targeted at folks who clone a shared code base for the first time, so they don't have the benefit of the execution from the generator or the instructions at the end. I thought this would be useful since there isn't a "standard" way to prepare and launch a Python project, unlike in the Java world. Developers should obviously edit the readme to suit the project, like if they change the base module name etc.